airflow/www/views.py

```diff
@@ -394,8 +394,10 @@ def health(self):
         return wwwutils.json_response(payload)
 
     @expose('/home')
-    @has_access
-    def index(self):  # pylint: disable=too-many-locals,too-many-statements
+    @auth.has_access([
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_WEBSITE),
+    ])  # pylint: disable=too-many-locals,too-many-statements
```

Comment:
Doesn't the pylint comment want to be on the next line?

Reply:
Nope. See the other comment. I'm not sure why’d , but it throws it at this level for some reason. I presume it has to do with the reassigning the view function data to the output of the wrapper function.